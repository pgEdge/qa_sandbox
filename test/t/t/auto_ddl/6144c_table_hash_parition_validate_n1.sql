-- This file runs on n1 again to see all the table and their partitions have been dropped on n1 (as a result of drop statements)
-- being auto replicated via 6144b

SELECT pg_sleep(1);--to ensure all objects are replicated

SET ROLE appuser;

SET search_path TO s614, public;
-- none of these tables should exist.
\d sales_hash_1
\d sales_hash_2
\d sales_hash_3
\d sales_hash_4
\d sales_hash
SELECT * FROM get_table_repset_info('sales_hash');

\d products_hash
\d products_hash_1
\d products_hash_2
\d products_hash_3
\d products_hash_4
SELECT * FROM get_table_repset_info('products_hash');

RESET ROLE;
--dropping the schema
DROP SCHEMA s614 CASCADE;