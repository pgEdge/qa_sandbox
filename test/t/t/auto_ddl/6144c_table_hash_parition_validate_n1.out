-- This file runs on n1 again to see all the table and their partitions have been dropped on n1 (as a result of drop statements)
-- being auto replicated via 6144b
SELECT pg_sleep(1);--to ensure all objects are replicated
 pg_sleep 
----------
 
(1 row)

SET ROLE appuser;
SET
SET search_path TO s614, public;
SET
-- none of these tables should exist.
\d sales_hash_1
Did not find any relation named "sales_hash_1".
\d sales_hash_2
Did not find any relation named "sales_hash_2".
\d sales_hash_3
Did not find any relation named "sales_hash_3".
\d sales_hash_4
Did not find any relation named "sales_hash_4".
\d sales_hash
Did not find any relation named "sales_hash".
SELECT * FROM get_table_repset_info('sales_hash');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d products_hash
Did not find any relation named "products_hash".
\d products_hash_1
Did not find any relation named "products_hash_1".
\d products_hash_2
Did not find any relation named "products_hash_2".
\d products_hash_3
Did not find any relation named "products_hash_3".
\d products_hash_4
Did not find any relation named "products_hash_4".
SELECT * FROM get_table_repset_info('products_hash');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

RESET ROLE;
RESET
--dropping the schema
DROP SCHEMA s614 CASCADE;
INFO:  DDL statement replicated.
DROP SCHEMA
